src/Game.test.js

```diff
@@ -0,0 +1,154 @@
+
+const {
+    isLosingMove,
+    isWinningMove,
+    isGameOver,
+    playCard,
+    aboutToPlay,
+    cancelPlay,
+    resetGame
+} = require('./Game')
+
+const G = {
+    players: {
+        1: {
+            cards: [67, 92]
+        },
+        2: {
+            cards: [45, 99]
+        }
+    },
+    aboutToPlay: {
+        1: null
+    },
+    cardsPlayed: [3],
+}
```

Comment:
Why is this global? We should be able to write all of the tests in a self-contained way. Any references to `G` within the tests should be unnecessary. 

For example, the test right below could change use 

```
const players = { ... };
expect(isLosingMove(cardJustPlayed, players)).toBe(true);
```

instead of

```
G.players = { ... };
expect(isLosingMove(cardJustPlayed, G.players)).toBe(true);
```